SimTKcommon/Geometry/include/SimTKcommon/internal/DecorativeGeometry.h

```diff
@@ -97,7 +97,8 @@ DecorativeGeometry& operator=(const DecorativeGeometry& source);
 
 /** Drawing modes. **/
 enum Representation {
-    DrawPoints    =  1, ///< Use a cloud of points.
+    Hide = 0, ///< hide geometry for now to display later
```

Comment:
@sherm1 If display preference in the GUI is changed to Hide it's less overhead if the DecorativeGeometry is marked as Hidden rather than completely removed. It also makes our serializable Geometry::Representation enum consistent with Simbody's counterpart.